server/app/views/components/AlertFragment.html

```diff
@@ -32,7 +32,13 @@
       th:text="${alertSettings.title.get()}"
     ></h6>
 
-    <p class="usa-alert__text" th:text="${alertSettings.text}"></p>
+    <!-- TODO ssandbekkhaug fix why both texts appear -->
+    <div th:if="${alertSettings.unescapedDescription()} == true">
+      <p class="usa-alert__text" th:utext="${alertSettings.text}"></p>
```

Comment:
Blocking - Should we add a class "bg-red-600" to make sure its red? May be it is a question for @sijiayam